src/dfi/mn_checks.cpp

```diff
@@ -1155,7 +1155,8 @@ Res CPoolSwap::ExecuteSwap(CCustomCSView &view,
     result = swapAmountResult.nValue;
 
     // Send final swap amount Rust side for indexer
-    if (txInfo) {
+    bool isOceanEnabled = gArgs.GetBoolArg("-oceanarchive", false);
```

Comment:
due to circular dep caused.. hardcoded the default as false as similar with `static constexpr bool DEFAULT_OCEAN_ARCHIVE_ENABLED = false`

Reply:
Sounds good to me, we'll resolve it correctly later